commands/elevatorsetting.py

```diff
@@ -81,6 +80,32 @@ def execute(self) -> None:
         self.elevator.setIntakePosition()
 
 
+class ElevatorIntakePositionToggleOn(Command):
+    def __init__(self, elevatorSubsystem: ElevatorSubsystem) -> None:
+        Command.__init__(self)
+        self.setName(__class__.__name__)
+        self.elevator = elevatorSubsystem
+
+    def execute(self) -> None:
+        self.elevator.setDefaultCommand(ElevatorIntakePosition(self.elevator))
```

Comment:
i mean i guess, i do not like how scuffed this is for default command override but hey, if it works it works